dev/dev/devDeployment.yaml

```diff
@@ -91,6 +90,7 @@ oidc-proxy:
     ENABLE_STACKDRIVER: yes
     FILTER2: AddOutputFilterByType DEFLATE application/json text/plain text/html application/javascript
       application/x-javascript
+    CORS_REGEX: \\Qhttps://jade.datarepo-dev.broadinstitute.org\\Egit che
```

Comment:
I think there might be some copypasta at the end of this line!

Reply:
gah, yeah...derp

Reply:
had the wrong window selected 🤦